### ISSUE TITLE
Conflicting option in `snow function create -h`.

### ISSUE THREAD

**Issue by ugmuka** [Snowflake-Labs/snowcli#68]
I executed missing option in `snow function create`.
```
$ snow function create aaa
Usage: snow function create [OPTIONS]
Try 'snow function create -h' for help.
╭─ Error ───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
│ Missing option '--name' / '-n'.                                                                                                                                                                                                           │
╰───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
```

However, `snow function create -h` is failed.

```
$ snow function create -h
╭─ Error ───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
│ Option '-h' requires an argument.                                                                                                                                                                                                         │
╰───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
```
It is because `-h` option is interpreted as `--handler` option.
The message in error should be `snow function create --help`, not `snow function create -h`.

**Comment by sfc-gh-jhollan** [comment]
Great point - yes I've wondered if we should even use a different flag for handler because I've hit this same one too. Thanks for flagging

**Comment by sfc-gh-jsikorski** [comment]
Should be fixed with #438 - will be released in next version
